### ISSUE TITLE
Extract list of contributors from the changelog

### ISSUE THREAD

**Issue by albe** [neos/BuildEssentials#54]
As suggested by @markusguenther :house_with_garden: on slack:

> 11:36 I really like that WordPress is highlighting the contributors of the new versions so nice… maybe also a thing for us :slightly_smiling_face:
![image](https://user-images.githubusercontent.com/4259532/117144447-3a5eb000-adb2-11eb-8e2d-09410941e71b.png)

We could extend the `CreateChangelog` [command](https://github.com/neos/BuildEssentials/blob/7.1/build-tools.php#L214) (or create a new one) to  extract the list of the contributors and add it to the end of the log. Then the RM can mention those in the release blog post, which would create a bit of an accomplishment and maybe even something to be proud of for the contributors. At least it was for me back when I got my first contribution into a release.

**Comment by kdambekalns** [comment]
We could also use https://allcontributors.org/

**Comment by kdambekalns** [comment]
Or something really simple to start from: `git log --pretty="%aN" ${VERSION}...${PREVIOUS_VERSION} | sort | uniq`

**Comment by albe** [comment]
> Or something really simple to start from: `git log --pretty="%aN" ${VERSION}...${PREVIOUS_VERSION} | sort | uniq`

Yeah, that was roughly along the lines I was thinking of as a first step.
